frontend/src/features/admin-form/create/logic/components/LogicContent/EditLogicBlock/EditCondition/ThenShowBlock.tsx

```diff
@@ -70,13 +74,67 @@ export const ThenShowBlock = ({
       : 'show'
   }, [logicTypeValue])
 
+  const [deletedFieldsCount, setDeletedFieldsCount] = useState<number>(0)
```

Comment:
is the `<number>` not inferred?

Reply:
looks like we have a mix of `useState(0)` and `useState<number>(0)` haha, I prefer the former

Reply:
sure, i like types but type inference is also cool